lib/interface_quda.cpp

```diff
@@ -510,6 +512,68 @@ void initQudaDevice(int dev)
   profileInit.TPSTOP(QUDA_PROFILE_TOTAL);
 }
 
+void printDeviceProp()
```

Comment:
This whole function should be moved to the `target/cuda/device.cpp` file, and called with something like  `device::print_prop()`.